src/Dependencies/Collections/SegmentedDictionary`2.cs

```diff
@@ -442,7 +442,7 @@ public Enumerator GetEnumerator()
 ConcurrentOperation:
             ThrowHelper.ThrowInvalidOperationException_ConcurrentOperationsNotSupported();
 ReturnFound:
-            ref var value = ref entry._value;
+            ref TValue value = ref entry._value;
```

Comment:
It wasn't clear to me why this needed to change.

Reply:
Wouldn't bootstrap without this change